vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-# Business Objective
+---
+sidebarDepth: 2
+---
 
-The Entando Process Driven Applications plugin is built to provide a
-rich and full featured user experience to allow for the management and
-completion of business processes and automation. The UX is delivered a
-general purpose UX layer built using micro frontends that can be
-utilized for any business process or task engine. Customers drive the UX
-layer by utilizing a pre-developed Entando integration or by
-implementing a set of interfaces on the server side. The backend is a
-Spring Boot microservice that provides a pluggable interface allowing
-for the injection of any underlying process or automation toolkit. The
-interfaces and steps for creating a new PDA backend implementation are
-provided below. This document provides an overview of the Entando
-Process Driven Apps (PDA) plugin. The initial section covers the
-business functionality and user facing micro frontends. The second
-section covers the architecture and the points of extension for the PDA
-plugin. The third provides a detailed technical overview for developers
-to add new functionality to a PDA implementation.
+# Entando Process Driven Applications Plugin
 
-# Terminology
+## Introduction
 
-## Micro Frontend
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  The solution template includes: 
 
-A Micro frontend architecture is an approach to developing web
-application as a composition of small frontend apps. Instead of writing
-a large monolith frontend application, the application is broken down
-into domain specific micro frontends, which are self-contained and can
-be developed and deployed independently.
+- A general purpose UX layer created from micro frontends and can be implemented for any business process or task engine. The UX layer can serve data via the included Entando integration adapter or by implementing a set of interfaces on the server side.
+- A Spring Boot microservice backend providing a pluggable interface for the injection of underlying processes or automation toolkits. The interfaces and steps for creating a new PDA backend implementation are described in further detail [here](link).
 
-## Process Automation Manager (PAM)
+This tutorial will guide you through installing the PDA plugin using the Entando Component Repository (ECR) and an Entando bundle. The goal of this exercise is to demonstrate the scope of process automation functionality enabled by integrating the following components: 
 
-A business process automation engine built and maintained by Red Hat
+- custom Micro Frontends
```

Comment:
1) Capitalize each item if any are capitalized. 
2) We should probably use the full product name - Red Hat Process Automation Manager and link to it - https://www.redhat.com/en/technologies/jboss-middleware/process-automation-manager
3) 'Backend for Frontend (BFF) microservice' would be cleared.

Reply:
This list seems to duplicate the first paragraph as I read it again. Maybe we don't need the list too?

Reply:
@nshaw as a non-dev the duplication isn't totally obvious... can we leave it in for laymen to drive the message home? there is no explicit repetition, and this bullet style in the intro is something jinah and i have been trying to be consistent with...

will make a note to capitalize all entries in lists/bullets, not just proper nouns

Reply:
@Lyd1aCla1r3 , that's fine. I might move the Red Hat PM entry into last place then but I'm not sure why it feels better, maybe because it's an external integration.